src/sentry/api/serializers/models/release.py

```diff
@@ -428,7 +432,7 @@ def get_attrs(self, item_list, user, **kwargs):
 
         if environments is None:
```

Comment:
rather than passing no_snuba into this helper, can we simply skip to the default below if `no_snuba` is set?

otherwise `first_seen` and `last_seen` will be missing and may affect the response as well